githooks/pre-commit

```diff
@@ -8,12 +8,3 @@ if [ $? -ne 0 ]; then
  exit 1
 fi
 
-
-echo "Adding license header."
-./license-add.sh
```

Comment:
We probably want to remove this file as well.

Reply:
Hi, thanks for the notice. I removed it now.